tests/test_manifests.py

```diff
@@ -25,14 +25,22 @@ def test_product_manifest_properties():
     product = ProductFactory(product_variants=["1"])
     version = ProductVersionFactory(product_variants=["1"])
     stream = ProductStreamFactory(product_variants=["1"])
+    stream.save()
```

Comment:
You don't need to call .save() here, the factory creates the model instance for you.

I don't know why the test fails, but you've changed how the manifest is generated so you probably also need to change what the test is checking for in the various assertions.  You can run the tests with -vvvv to get debug output that shows the full diff between what was asserted and was actually generated. The error on Gitlab is:
```
        # One component linked to this product
        num_components = len(variant.components)
        assert num_components == 1
        # Manifest contains info for all components + the product itself
>       assert len(manifest["packages"]) == 2
E       AssertionError: assert 1 == 2
```

"len(variant.components)" probably needs to change to "len(variant.get_latest_components)" to match the manifest template. If we're generating the manifest using get_latest_components, I would check to make sure that filter is including the test component in its output.

Once you fix the above len() statement, I bet "assert num_components == 1" will start failing too, AKA we return the test component in the full list of components but not in the filtered list. In other words, I think this is probably a bug in the filter (or a bug in how we generate the test data), and not a bug in the manifest template itself.